File: chrome/test/data/extensions/platform_apps/music_manager_private/device_id_value_returned/chrometest.js
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
// Copyright 2013 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

chrome.test.runTests([
  function getDeviceIdTest() {
    chrome.musicManagerPrivate.getDeviceId(function(id) {
      if (id !== undefined) {
        chrome.test.assertNoLastError();
        chrome.test.assertEq('string', typeof id);
        chrome.test.assertTrue(id.length >= 8);
      } else {
        // Bots may not support device ID. In this case, we still check that the
        // error message is what we expect.
        chrome.test.assertLastError(
            'Device ID API is not supported on this platform.');
      }
      chrome.test.succeed();
    });
  }
]);

File: chrome/test/chromeos/autotest/files/client/deps/chrome_test/chrome_test.py
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
#!/usr/bin/env python3

# Copyright (c) 2010 The Chromium OS Authors. All rights reserved.
# Use of this source code is governed by a BSD-style license that can be
# found in the LICENSE file.

import common, os
from autotest_lib.client.bin import utils

version = 1

def setup(top_dir):
    return


pwd = os.getcwd()
utils.update_version(pwd + '/src', False, version, setup, None)
